ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -6,7 +6,7 @@ public class DomainRandomization : MonoBehaviour
 {
     public GameObject table;
     public GameObject cube;
-    public GameObject robot;
+    //public GameObject robot;
```

Comment:
Did you also uncheck the robot in the scene? I don't see any evidence of that in this diff. Or are you just not moving it?

Reply:
I removed it from the scene